content/participants/ttjc@Harshith7016/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Harshith7016
+type: "participant"
+name: "Harshith Venkatesh"
+portfolio: "https://Harshi7016.github.io"
+github-link: "https://github.com/Harshi7016"
+linkedin-link: "https://www.linkedin.com/in/harshith-v-7016ece/"
+looking-for: "job"
+description: "TypeScript | Angular | React | 2 years experiance | Exploring Opportunities"
+---
+
+Namaste, I am Harshith Venkatesh , let me walk you through my intro, projects and blogs.
+“When you stop expecting people to be perfect, you can like them for who they are.”
+
+### introduction
+
+I am currently working in CenturyLink as a Software Engineer holding accountability to the entire user interface module for the project I am currently working on.
+
+I have pursued B.Tech - Electronics and Communication branch in Manipal Institute of Technology and transitioned from Telecommunication Domain to Information Technology Domain in the span of One and Half year and I really enjoy exploring different opportunities that IT provides.
+
+## why are you taking this challenge?
+
+Honestly, I really liked the challenge and this will definitely motivate me to explore different flavours of JS .
+
+I would really want to be part of Open Source community.
+
+I believe working on typescript for 1.5 years has built enough confidence in me to create a project in Angular Framework from Scratch. I would like to build my portfolio in a way to attract companies to look into my portfolio and thus leading to Switching for better opportunities.
+
+## something interesting about you?
+
+I am ambidextrous ,I believe I am self taught front end developer.
+
+I spend my weekends teaching kids and helping communities by taking teaching drives and thus helping in developing the community.
+
+### help I need?
+
+It would be good if someone helps and suggest me the best practices while working on projects.
+Maybe just pointing me out that a particular link in youtube would help us more on particular tech stack would definately make some one's day better :)
```

Comment:
Hey! @Harshi7016 a small mistake here:
* definately

Reply:
Hey, I really appreciate that you have gone through what I have written and made suggestions